packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -251,6 +275,27 @@ describe(Runner, () => {
         await runner.run();
     });
 
+    it('return redirected url', async () => {
+        setupWebDriverCalls();
+
+        setupUpdateScanRunResultCall(getRunningJobStateScanResult());
+
+        const clonedPassedAxeScanResults = cloneDeep(passedAxeScanResults);
+        clonedPassedAxeScanResults.scannedUrl = 'redirect url';
+        scannerTaskMock
+            .setup(async s => s.scan(scanMetadata.url))
+            .returns(async () => Promise.resolve(clonedPassedAxeScanResults))
+            .verifiable();
```

Comment:
extract this to a helper method - setupVerfiableScanCall(scannedUrl)
(scanned url will be scanMetadata.url if not set)

Reply:
not worth it, it's only one test